mPower2/mPower2/Medication.json

```diff
@@ -0,0 +1,140 @@
+              },
+              {
+              "sectionIdentifier": "Amantadine",
+              "identifier": "Extended release amantadine",
+              "detail": "(Gocovri)"
+              },
+              {
+              "sectionIdentifier": "Anticholinergic Medications",
+              "identifier": "Artane",
+              "detail": "(Trihexyphenidyl)"
+              },
+              {
+              "sectionIdentifier": "Anticholinergic Medications",
+              "identifier": "Cogentin",
+              "detail": "(Benztropine)"
+              },
+              {
+              "sectionIdentifier": "Catechol-O-methyltransferase inhibitors",
+              "identifier": "Comtan",
+              "detail": "(Entacapone)"
+              },
+              {
+              "sectionIdentifier": "Catechol-O-methyltransferase inhibitors",
+              "identifier": "Stalevo",
+              "detail": "(Levodopa / Carbidopa and Entacapone)"
+              },
+              {
+              "sectionIdentifier": "Catechol-O-methyltransferase inhibitors",
+              "identifier": "Tasmar",
+              "detail": "(Tolcapone)"
+              },
+              {
+              "sectionIdentifier": "Dopamine promotor",
+              "identifier": "Apokyn",
+              "detail": "(Apomorphine)"
+              },
+              {
+              "sectionIdentifier": "Dopamine promotor",
+              "identifier": "Mirapex ER",
+              "detail": "(Pramipexole)"
+              },
+              {
+              "sectionIdentifier": "Dopamine promotor",
+              "identifier": "Neupro",
+              "detail": "(Rotigotine)"
+              },
+              {
+              "sectionIdentifier": "Dopamine promotor",
+              "identifier": "Requip",
+              "detail": "(Ropinrole)"
+              },
+              {
+              "sectionIdentifier": "Dopamine promotor",
+              "identifier": "Requip XL",
+              "detail": "(Ropinrole)"
+              },
+              {
+              "sectionIdentifier": "Dopamine replacement therapy",
+              "identifier": "Duopa",
+              "detail": "(Carbidopa / Levodopa)"
+              },
+              {
+              "sectionIdentifier": "Dopamine replacement therapy",
+              "identifier": "Parcopa",
+              "detail": "(Carbidopa / Levodopa)"
+              },
+              {
+              "sectionIdentifier": "Dopamine replacement therapy",
+              "identifier": "Sinemet",
+              "detail": "(Carbidopa / Levodopa)"
+              },
+              {
+              "sectionIdentifier": "Dopamine replacement therapy",
+              "identifier": "Sinemet CR",
+              "detail": "(Carbidopa / Levodopa)"
+              },
+              {
+              "sectionIdentifier": "Dopamine replacement therapy",
+              "identifier": "Stalevo",
+              "detail": "(Carbidopa / Levodopa)"
```

Comment:
Stalevo is listed in a different section above with (Levodopa / Carbidopa and Entacapone)

Reply:
Yes, I know. ;)